src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -86,6 +86,7 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
 
     /** Constants for 'Add existing questions' and 'Import file' features **/
     showExistingQuestions = false;
+    showExistingQuestionsFromAnotherCourse = false;
```

Comment:
Instructors can import questions from the same or other courses.
The name is a bit misleading.

Reply:
```suggestion
    showExistingQuestionsFromCourse = false;
```